### DISCUSSION TITLE
fwiw note around umbrel install

### DISCUSSION THREAD

**Discussion by statespacedev** [bitcoinknots/bitcoin#57] [General]
hi luke - thanks to jimmy song's newsletter, got to this page yesterday - https://wtfhappenedinfeb2023.com/install.html - immediately followed those instructions for switching umbrel from core to knots

have a 1tb disk and noticed this morning usage had gone from about 750gb to 950gb - looked like the old core data folders were still there, so deleted those. not sure where things are now, going to wait a bit and see if everything turns out ok - otherwise may end up redoing that box from scratch with knots.

**Comment by luke-jr** [comment]
Note that higher memory usage (not disk usage) is to be expected: by default, Knots will use extra/unused memory to optimise performance, but releases it as needed.

**Comment by statespacedev** [comment]
quick update - seemed like that umbrel box was having troubles after manually deleting the data folders for core

went ahead and just redid the box back to default ubuntu with default umbrel - ran the instructions from the link above again

running knots now - initial chain sync underway and looking normal
